### PR TITLE
Add krmayankk to kubernetes-milestone-maintainers team

### PR DESCRIPTION
**Pull request by krmayankk**

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ teams:
     - idvoretskyi # PM
     - nikhita # ContribEx
     - spiffxp # 1.14 RT Lead / Testing
+    - krmayankk # 1.14 RT Bug Triage
```
Comment (nikhita) [inline config/kubernetes/sig-release/teams.yaml]: this list needs to be sorted :P

Comment (nikhita) [reply config/kubernetes/sig-release/teams.yaml]: and you'll need to add in the members list

(only org admins can be in the maintainers list)

**Comment by justaugustus** [comment]
@krmayankk -- there is a maintainers listing for each team across all org configs in this repo, so this is a little confusing to a passerby.

Could you amend to the commit msg and issue title to something like `Add krmayankk to kubernetes-milestone-maintainers team`?

**Comment by krmayankk** [comment]
@justaugustus  done

**Comment by justaugustus** [comment]
@krmayankk please amend the commit message too

**Comment by krmayankk** [comment]
@justaugustus done

**Comment by justaugustus** [comment]
Thanks @krmayankk!
/lgtm
/approve

**Comment by krmayankk** [comment]
@justaugustus did i add it to the wrong place, my powers dont work yet :-( https://github.com/kubernetes/website/pull/12297

**Comment by justaugustus** [comment]
@krmayankk -- I believe there are currently some issues with peribolos (the tool the leverages these configs), so it's possible that the periodic GH org sync was not successful.

@nikhita / @spiffxp / @fejta might be able to provide more details for you.

**Comment by nikhita** [comment]
Yeah, peribolos is currently red right now...created https://github.com/kubernetes/org/pull/591 to help fix it.

**Comment by nikhita** [comment]
@krmayankk you should have powers to apply the milestone right now! :tada: :smile:

**Pull request merged**